lib/livebook_web/live/apps_live.ex

```diff
@@ -41,170 +41,184 @@ defmodule LivebookWeb.AppsLive do
+                </div>
+              </div>
+            </div>
+            <div class="border border-gray-200 rounded-b-lg overflow-auto tiny-scrollbar whitespace-none">
+              <%= if Enum.any?(app.sessions) do %>
+                <div class="uppercase text-gray-500 text-sm font-medium leading-normal tracking-wider px-4 pt-4 pb-3">
+                  Runing Sessions
+                </div>
+                <.grid rows={app.sessions}>
+                  <:col :let={app_session} label="Status">
+                    <a
+                      aria-label="debug app"
+                      href={app_session.app_status == :error && ~p"/sessions/#{app_session.id}"}
+                      target="_blank"
+                    >
+                      <.app_status status={app_session.app_status} />
+                    </a>
+                  </:col>
+                  <:col :let={app_session} label="Uptime">
+                    <%= format_datetime_relatively(app_session.created_at) %>
+                  </:col>
+                  <:col :let={app_session} label="Version">
+                    v<%= app_session.version %>
+                  </:col>
+                  <:col :let={app_session} label="Clients">
+                    <%= app_session.client_count %>
+                  </:col>
+                  <:actions :let={app_session}>
+                    <span class="tooltip left" data-tooltip="Open">
+                      <a
+                        class={[
+                          "icon-button",
+                          app_session.app_status.lifecycle != :active && "disabled"
+                        ]}
+                        aria-label="open app"
+                        href={~p"/apps/#{app.slug}/#{app_session.id}"}
+                      >
+                        <.remix_icon icon="link" class="text-lg" />
+                      </a>
+                    </span>
+                    <span class="tooltip left" data-tooltip="Debug">
+                      <a
+                        class="icon-button"
+                        aria-label="debug app"
+                        href={~p"/sessions/#{app_session.id}"}
+                      >
+                        <.remix_icon icon="terminal-line" class="text-lg" />
+                      </a>
+                    </span>
+                    <%= if app_session.app_status.lifecycle == :active do %>
+                      <span class="tooltip left" data-tooltip="Deactivate">
+                        <button
+                          class="icon-button"
+                          aria-label="deactivate app session"
+                          phx-click={
+                            JS.push("deactivate_app_session",
+                              value: %{slug: app.slug, session_id: app_session.id}
+                            )
+                          }
+                        >
+                          <.remix_icon icon="stop-circle-line" class="text-lg text-brand-pink" />
```

Comment:
I would keep it gray as it stands out too much

```suggestion
                          <.remix_icon icon="stop-circle-line" class="text-lg" />
```